.github/workflows/developer-site-ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Libra Developers Website CI
+
+on:
+  push:
+    branches: [ auto ]
+    paths: [ 'developers.libra.org/**' ]
+  pull_request:
+    branches: [ master ]
+    paths: [ 'developers.libra.org/**' ]
+
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checks-out the Libra website repository under $GITHUB_WORKSPACE, so job can access it
+    - uses: actions/checkout@v2
+    # Installs node and yarn
+    - name: Use Node.js 12
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    # Install git
+    - name: Install git
+      run: |
+        sudo apt --assume-yes update
+        sudo apt --assume-yes install git
+    # Test that building the site is successful
+    - name: Build Site
+      run: ./developers.libra.org/scripts/build_docs.sh -b
```

Comment:
This should also build the rust docs (pass `-r`) since we're removing the doc build from Circle